### PR TITLE
Update strings.po

### PR DESCRIPTION
**Pull request by theheroGAC**
translated 45 strings 👍

### CODE REVIEW

File: language/Italian/strings.po [hunk 1]
```diff
@@ -2475,11 +2475,11 @@ msgstr "Impostazioni"
 
 #: plugins/timer/init.lua:94
 msgid "Current time"
-msgstr ""
+msgstr "Ora attuale"
```
Comment (cuavas) [inline language/Italian/strings.po]: This is used for the the play time for the current session - is this translation appropriate?  (The English is not particularly clear for this string.)

File: language/Italian/strings.po [hunk 2]
```diff
@@ -2753,27 +2753,27 @@ msgstr ""
 
 #: plugins/cheatfind/init.lua:692
 msgid "Cheat engine not available"
-msgstr ""
+msgstr "Cheat engine non disponibile"
 
 #: plugins/cheatfind/init.lua:718
 msgid "Default name is "
-msgstr ""
+msgstr "Il nome predefinito è"
```
Comment (cuavas) [inline language/Italian/strings.po]: This is missing a trailing space (the string is concatenated with a name - this is bad form and will be fixed at some point, but for now please add the space).

Comment (theheroGAC) [reply language/Italian/strings.po]: ok thank you fix now

Comment (theheroGAC) [reply language/Italian/strings.po]: fixed 2 strings

**Comment by cuavas** [comment]
Thanks

**Comment by etabeta78** [comment]
On Sat, Dec 2, 2017 at 9:06 AM, Vas Crabb <notifications@github.com> wrote:

> *@cuavas* commented on this pull request.
> ------------------------------
>
> In language/Italian/strings.po
> <https://github.com/mamedev/mame/pull/2875#discussion_r154491066>:
>
> > @@ -2475,11 +2475,11 @@ msgstr "Impostazioni"
>
>  #: plugins/timer/init.lua:94
>  msgid "Current time"
> -msgstr ""
> +msgstr "Ora attuale"
>
> This is used for the the play time for the current session - is this
> translation appropriate? (The English is not particularly clear for this
> string.)
>

It might be more appropriate something along the lines of "Tempo trascorso"
or "Tempo attuale" (the former better than the latter, meaning "Elapsed
time").

my 2c
    Fabio

**Pull request merged**